editoast/src/railjson/operation/update.rs

```diff
@@ -108,6 +109,24 @@ mod test {
         length: f64,
     }
 
+    #[derive(QueryableByName)]
+    struct SightDistance {
+        #[sql_type = "Double"]
+        sight_distance: f64,
+    }
+
+    #[derive(QueryableByName)]
+    struct Comment {
+        #[sql_type = "Text"]
+        comment: String,
+    }
+
+    #[derive(QueryableByName)]
+    struct Speed {
+        #[sql_type = "Double"]
+        speed: f64,
+    }
+
```

Comment:
You can factor `Speed` and `SightDistance` structures like this:

```rust
struct Value {
    #[sql_type = "Double"]
    val: f64,
}
```

Then you can tweak your sql queries to rename the column `val` instead of `speed` or `sight_distance`...